cypress.js

```diff
@@ -0,0 +1,15 @@
+const {ERROR} = require('./config');
+
+module.exports = {
+  rules: {
+    'no-restricted-properties': [
+      ERROR,
+      {
+        object: 'it',
+        property: 'only',
+        message:
+          'This should only be used for debugging and removed afterwards.'
+      }
+    ]
```

Comment:
Very basic for now, but this was the only error I've ran so far into with Cypress that I think can be linted against.

There's https://github.com/cypress-io/eslint-plugin-cypress, but I don't see practical use for the proposed rules there as we're already following these rules quite consistently and if we make an exception, we typically document it anyway. Seems unnecessary to disable eslint additionally in such cases.